Model.cpp

```diff
@@ -129,6 +137,15 @@ int Model::insertBatch(vector<Model*> models, int batchsize, Mode mode)
                         conn.setString(paramNum, i->getValue());
                     }
                 }
+                else if(f->getType() == BOOLEAN)
```

Comment:
Actually, now is as good a time as any to do this. Convert this whole if statement to a switch and perform the null check before the switch.